index.js

```diff
@@ -136,6 +136,27 @@ exports.array = function(val) {
 };
 
 /**
+ * Check if an array contains only items from another array
+ * @param  {Array} arrayOfAcceptedValues
+ * @return {Function}
+ */
+exports.arrayContaining = function(arrayOfAcceptedValues) {
+    return function(val) {
+      var bool = true;
+      if(val && val instanceof Array) {
```

Comment:
`Array.isArray` is supposedly a better option than `instanceof` for checking arrays https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray

Its supported from >IE9 http://caniuse.com/#search=isArray